### PR TITLE
nodejs-fr: init french locale

### PR DESCRIPTION
**Pull request by clakech**
Hello all,

Why not a french locale for nodejs.org website to help frenchies to start ?

### CODE REVIEW

File: locale/fr/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  tagline-current: Dernières Fonctionnalités
+  tagline-lts: Recommandé pour la plupart des utilisateurs
+  changelog: Journal des modifications
+  api: API Docs
+  version-schedule-prompt: Ou regardez le
+  version-schedule-prompt-link-text: Planning LTS
+---
+
+Node.js® est un environnement d'exécution JavaScript construit sur le [moteur JavaScript V8 de Chrome](https://developers.google.com/v8/).
+Node.js utilise un modèle basé sur l'événementiel et des entrées sorties non bloquantes ce qui le rend
```
Comment (geoiris) [inline locale/fr/index.md]: entrées/sorties ?

**Review by geoiris — APPROVED** [review]

File: locale/fr/site.json [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+            "link": "foundation/tsc",
+            "text": "Gouvernance Technique"
+        },
+        "itn": {
+             "link": "foundation/in-the-news",
+             "text": "A la une"
+        },
+        "announce": {
+            "link": "foundation/announcements",
+            "text": "Annoncements"
```
Comment (lborie) [inline locale/fr/site.json]: Annonces

Comment (clakech) [reply locale/fr/site.json]: merki

File: locale/fr/index.md (hunk 1 above)
Comment (clakech) [reply locale/fr/index.md]: merki

File: locale/fr/site.json (hunk 2 above)
Comment (clakech) [reply locale/fr/site.json]: il me faut un petit 👍 en réaction sur la PR please ^^

File: locale/fr/index.md (hunk 1 above)
Comment (clakech) [reply locale/fr/index.md]: il me faut un petit 👍 en réaction sur la PR please ^^ @geoiris

**Review by lborie — APPROVED** [review]

**Review by vdeturckheim — DISMISSED** [review]

**Comment by vdeturckheim** [comment]
cc @nodejs/nodejs-fr

File: locale/fr/site.json [hunk 3]
```diff
@@ -0,0 +1,174 @@
+{
+    "title": "Node.js",
+    "author": "Node.js Foundation",
+    "url": "https://nodejs.org/fr/",
+    "locale": "fr",
+    "scrollToTop": "Faire définler en haut",
```
Comment (abenhamdine) [inline locale/fr/site.json]: défiler

**Review by abenhamdine — CHANGES_REQUESTED** [review]

File: locale/fr/site.json (hunk 3 above)
Comment (clakech) [reply locale/fr/site.json]: thanks

Comment (clakech) [reply locale/fr/site.json]: fixed && pushed

File: locale/fr/index.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  tagline-current: Dernières Fonctionnalités
+  tagline-lts: Recommandé pour la plupart des utilisateurs
+  changelog: Journal des modifications
+  api: API Docs
+  version-schedule-prompt: Ou regardez le
+  version-schedule-prompt-link-text: Planning LTS
+---
+
+Node.js® est un environnement d'exécution JavaScript construit sur le [moteur JavaScript V8 de Chrome](https://developers.google.com/v8/).
+Node.js utilise un modèle basé sur l'événementiel et des entrées/sorties non bloquantes, ce qui le rend léger et efficace. L'écosystème de logiciels de Node.js, [npm](https://www.npmjs.com/), est le plus grand écosystème de librairies open source au monde.
```
Comment (harf18) [inline locale/fr/index.md]: librairies => bibliothèques ?

**Review by harf18 — CHANGES_REQUESTED** [review]

File: locale/fr/index.md (hunk 4 above)
Comment (clakech) [reply locale/fr/index.md]: Both are ok, what others think about that?

Comment (GromNaN) [reply locale/fr/index.md]: "librairies" for code libraries is an Anglicism that sounds very familiar, like "open source".

File: locale/fr/site.json [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+            "link": "feed/releases.xml",
+            "text": "Blog de Node.js: Versions"
+        },
+        {
+            "link": "feed/vulnerability.xml",
+            "text": "Blog de Node.js: Rapport de Vulnérabilité"
+        },
+        {
+            "link": "feed/tsc-minutes.xml",
+            "text": "Compte rendus des réunions TSC Node.js"
```
Comment (GromNaN) [inline locale/fr/site.json]: "Compte-rendus"
https://fr.wiktionary.org/wiki/compte-rendus

File: locale/fr/site.json [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+        }
+    ],
+    "home": { "text": "Accueil" },
+    "about": {
+        "link": "about",
+        "text": "A propos",
```
Comment (GromNaN) [inline locale/fr/site.json]: "À propos"

File: locale/fr/site.json [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+            "link": "foundation/board",
+              "text": "Comité"
+        },
+        "tsc": {
+            "link": "foundation/tsc",
+            "text": "Gouvernance Technique"
+        },
+        "itn": {
+             "link": "foundation/in-the-news",
+             "text": "A la une"
```
Comment (GromNaN) [inline locale/fr/site.json]: "À la Une"

File: locale/fr/site.json [hunk 8]
```diff
@@ -0,0 +1,174 @@
...
+        },
+        "announce": {
+            "link": "foundation/announcements",
+            "text": "Annonces"
+        },
+        "education": {
+            "link": "foundation/education",
+            "text": "Initiatives Educatives"
```
Comment (GromNaN) [inline locale/fr/site.json]: "Initiatives éducatives"

File: locale/fr/site.json [hunk 9]
```diff
@@ -0,0 +1,174 @@
...
+          "link": "get-involved/events",
+          "text": "Evènements"
+        },
+        "conduct": {
+            "link": "https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#code-of-conduct",
+            "text": "Code de Conduite"
+        }
+    },
+    "security"   : { "link": "security",        "text": "Securité" },
+    "blog"       : { "link": "blog",            "text": "Nouvelles" },
```
Comment (GromNaN) [inline locale/fr/site.json]: "Actualités"

**Review by GromNaN — COMMENTED** [review]
Not using accents on capital letters is considered as an orthographic mistake.

**Comment by vdeturckheim** [comment]
I dismissed my review based on @GromNaN comments.

File: locale/fr/site.json (hunk 5 above)
Comment (cedced19) [reply locale/fr/site.json]: It's a misspelling: "Comptes rendus"

File: locale/fr/index.md (hunk 4 above)
Comment (cedced19) [reply locale/fr/index.md]: I'm agree with @harf18. "bibliothèques" is the correct form.
If we have a word to translate english in a easy way, we should use this.
"open source" can be translate to "code source ouvert" but in some cases this translation is bad.
**To put it in a nutshell "open source"≠"code source ouvert" but "librairies" = "bibliothèques"**.

Comment (clakech) [reply locale/fr/index.md]: fixed && pushed @cedced19  @GromNaN @harf18

File: locale/fr/site.json (hunk 5 above)
Comment (clakech) [reply locale/fr/site.json]: fixed && pushed

File: locale/fr/site.json (hunk 6 above)
Comment (clakech) [reply locale/fr/site.json]: fixed && pushed

File: locale/fr/site.json (hunk 7 above)
Comment (clakech) [reply locale/fr/site.json]: fixed && pushed

File: locale/fr/site.json (hunk 8 above)
Comment (clakech) [reply locale/fr/site.json]: fixed && pushed

File: locale/fr/site.json (hunk 9 above)
Comment (clakech) [reply locale/fr/site.json]: fixed && pushed

**Comment by clakech** [comment]
Add a commit with correction from reviewers requested changes @vdeturckheim

File: locale/fr/site.json [hunk 10]
```diff
@@ -0,0 +1,174 @@
...
+            "link": "get-involved/development",
+            "text": "Développement"
+        },
+        "events": {
+          "link": "get-involved/events",
+          "text": "Evènements"
+        },
+        "conduct": {
+            "link": "https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#code-of-conduct",
+            "text": "Code de Conduite"
```
Comment (FKSI) [inline locale/fr/site.json]: Code de conduite

File: locale/fr/site.json [hunk 11]
```diff
@@ -0,0 +1,174 @@
...
+          "link": "SHASUMS256.txt.asc",
+          "text": "SHASUMS signés pour les fichiers des versions"
+        }
+    },
+    "docs": {
+        "link": "docs",
+        "text": "Docs",
+        "es6": {
+            "link": "docs/es6",
+            "text": "ES6 et au delà"
```
Comment (FKSI) [inline locale/fr/site.json]: au-delà

Comment (clakech) [reply locale/fr/site.json]: Done

File: locale/fr/site.json (hunk 10 above)
Comment (clakech) [reply locale/fr/site.json]: Done

**Review by cedced19 — APPROVED** [review]

**Review by FKSI — APPROVED** [review]

**Review by harf18 — APPROVED** [review]

File: locale/fr/site.json [hunk 12]
```diff
@@ -0,0 +1,174 @@
...
+            "link": "get-involved/contribute",
+            "text": "Contribuer"
+        },
+        "development": {
+            "link": "get-involved/development",
+            "text": "Développement"
+        },
+        "events": {
+          "link": "get-involved/events",
+          "text": "Evènements"
```
Comment (slester) [inline locale/fr/site.json]: -> Évènements

File: locale/fr/site.json [hunk 13]
```diff
@@ -0,0 +1,174 @@
...
+            "link": "foundation/members",
+            "text": "Membres"
+        },
+        "board": {
+            "link": "foundation/board",
+              "text": "Comité"
+        },
+        "tsc": {
+            "link": "foundation/tsc",
+            "text": "Gouvernance Technique"
```
Comment (slester) [inline locale/fr/site.json]: Ici (et autre part) le deuxième mot est en majuscule, mais en bas vous avez "Initiatives éducatives" -- il faut les harmoniser.

File: locale/fr/index.md [hunk 14]
```diff
@@ -0,0 +1,21 @@
+---
+layout: index.hbs
+labels:
+  current-version: Version Actuelle
+  download: Téléchargements
+  download-for: Téléchargements pour
+  other-downloads: Autres Téléchargements
```
Comment (slester) [inline locale/fr/index.md]: Majusculisation irrégulière (voir le commentaire en bas)

File: locale/fr/index.md [hunk 15]
```diff
@@ -0,0 +1,21 @@
...
+  other-lts-downloads: Autres Téléchargements LTS
+  other-current-downloads: Autres Téléchargements Actuels
+  current: Actuelle
+  lts: LTS
+  tagline-current: Dernières Fonctionnalités
```
Comment (slester) [inline locale/fr/index.md]: Majusculisation irrégulière partout dans ce fichier (voir le commentaire en bas)

**Review by slester — CHANGES_REQUESTED** [review]

**Comment by clakech** [comment]
@slester thanks, pushed

**Comment by fhemberger** [comment]
@GromNaN @slester @abenhamdine @vdeturckheim Can you please set a 👍 /👎  for your review, so we know if this change is good to merge? Thx!

**Review by abenhamdine — APPROVED** [review]

**Comment by slester** [comment]
Oops, sorry, I thought I did!

**Review by slester — APPROVED** [review]

**Comment by clakech** [comment]
7 reviewers approval, 6 👍 and even 1 🎉. Seems like we will have a nodejs.org french website very soon 😀

**Comment by fhemberger** [comment]
Incredible, merci bien à tous!

**Pull request merged**